minutes/minutes-2015-03-12.md

```diff
@@ -0,0 +1,368 @@
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        1. Draft charge has been made available to committee.
+
+        2. Mailing list created for this subcommittee: finance@lists.software-carpentry.org 
+
+* Partner Relations / Development
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        3. We have met and are working on the draft charge, to be approved by the subcommittee by 3/20. 
+
+        4. Ideas are being collected by Trello for final prioritization on 3/20.
+
+        5. Tom Wright has agreed to take on one of the subcommittee leadership roles, we are still working on having two ‘leaders’; Chair and Secretary 
+
+        6. Mailing list created for this subcommittee: development@lists.software-carpentry.org 
+
+* Assessment
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        7. Leadership roles have been determined for this committee, Daniel Chen as Chair, Jeramia Ory as Secretary (through July 2015) 
+
+        8. Daniel and Jeramia will now meet with Jason and Jory S. (+ others?) to prioritize collected ideas and reconvene the whole committee. Has created draft charge.
+
+        9. Mailing list revived: assessment@lists.software-carpentry.org
+
+	
+
+* Mentorship
+
+    * Liaison: Raniere Silva
+
+    * Update: Sheldon and Rayna has run an instructor debrief meeting.
+
+* Lesson Organization and Development
+
+    * Liaison: Matt Davis
+
+* Communication
+
+    * Liaison: Adina Chuang Howe
+
+    * Update:
+
+        10. Assigned:  brochure, badge, blogposts, description of workshops, press release template, summarizing steering committee activities, grant text
+
+        11. Open issues with no assignments:  twitter, annual report
+
+# ## Translations [ 10 min, until 0:45]
+
+* Lead: Greg
+
+* How do we deal with translated versions of the curriculum?
+
+# ## Corporate SWC Workshops [ 10 min, until 0:55 ]
+
+* Lead: Greg
+
+* There was a discussion about Monsanto and corporations in general having workshops.
+
+* Update from Greg on this topic
+
+* Anything we need to decide or deal with as SCF SC?
+
+# ## Issues in the board repo [ 10 min, until 1:10 ]
+
+Trevor has made some suggestions as some combination of issues and pull
+
+requests. I would like to vote on the ones that require board approval.
+
+* Lead: Katy
+
+* Public vs. private (sub)committee meetings [https://github.com/swcarpentry/board/issues/20](https://github.com/swcarpentry/board/issues/20)
+
+    * **Motion: Subcommittees should choose their operating procedures by themselves. Close this issue as "wontfix".**
+
+    * Passed
+
+* Revise 3-hour rule to include teaching [https://github.com/swcarpentry/board/pull/13](https://github.com/swcarpentry/board/pull/13)
+
+    * Trevor wants to include teaching hours as committee work. I and the interim steering committee felt oppositely when designing our current committee roles. The committee activities cannot be replaced by teaching activities. Otherwise, no committee work would get done.
```

Comment:
Actually, I want to include teaching hours in Software Carpentry activities (I agree that they aren't committee work).  Instead of trying to draw a line around committee work (including the nebulous “supporting initiatives prioritized by the Steering Committee”), I think it's better to say “you should shoot for three hours of general SWC contribution per week, [but your committee responsibilities come first][1]”.  See also [my earlier comment along these lines][2].  I think something along the lines of #13 would help clarify this, but I'm fine letting this cook until the committee has time to revisit it.

[1]: https://github.com/swcarpentry/board/commit/67eac1ef51eb46e411b0209c51551725ce3fdac6#diff-ad2ad3c7e1692c965cfada2b0d48a5afR190
[2]: https://github.com/swcarpentry/board/pull/13#issuecomment-78686649